server/manifest.go

```diff
@@ -7,5 +7,5 @@ var manifest = struct {
 	Version string
 }{
 	ID:      "zoom",
-	Version: "1.5.2-beta",
```

Comment:
have you generated this file or you have edited this because you should edit manifest file.

Reply:
I haven't edited this file. Actually I have changed the version in `plugin.json` and run a command `make dist` then this file gets generated automatically.